src/discordas.ts

```diff
@@ -214,20 +216,67 @@ async function run(): Promise<void> {
 
     roomhandler.bindThirdparty();
 
+    try {
+        await startDiscordBot(discordbot, appservice.botClient, config.bridge.adminMxid);
+        log.info("Discordbot started successfully");
+    } catch (err) {
+        log.error(err);
+        log.error("Failure during startup. Exiting");
+        process.exit(1);
+    }
+
     await appservice.begin();
     log.info(`Started listening on port ${port}`);
 
+}
+
+interface AdminRooms {
+    [mxid: string]: string;
+}
+
+async function notifyBridgeAdmin(client: MatrixClient, adminMxid: string, message: string) {
+    log.verbose(`Looking up admin room for ${adminMxid}`);
+    let adminRooms = await client.getAccountData<AdminRooms>(ADMIN_ROOMS_KEY).catch(() => null) || {};
+    if (!adminRooms[adminMxid]) {
+        log.verbose(`Creating an admin room for ${adminMxid}`);
+        adminRooms[adminMxid] = await client.createRoom();
+        await client.inviteUser(adminMxid, adminRooms[adminMxid]);
+        await client.setAccountData(ADMIN_ROOMS_KEY, adminRooms);
```

Comment:
Hah, clever. To avoid database usage?

Reply:
Yeah, I don't wanna add a whole new table for such a minor thing :)